### PR TITLE
Added script to collect tgt/pool pod logs for debugging

### PR DESCRIPTION
**Pull request by vishnuitta**
This PR adds a script that collect few details like PV/PVC/tgt/pool pod logs for an application.

Signed-off-by: Vishnu Itta <vitta@mayadata.io>

### CODE REVIEW

**Comment by ksatchit** [comment]
A minor change: can we move the scripts under debug-scripts/openebs/all_logs.sh - test-tools by nature consists of generic containers which can be used in a non-openebs context also, so this is to just help categorize.

File: debug-scripts/get_all_logs.sh [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	echo "Usage: bash get_all_logs.sh <application_pod> <application_pod_namespace> <output_file>"
+	echo "Note: This script doesn't do any kind of input validation, and, also, doesn't modify the cluster"
+	exit 1
+}
+
+get_kubectl_cmds_output()
+{
+	app_pod=$1
+	app_ns=$2
+	operator_ns="openebs"
```
Comment (kmova) [inline debug-scripts/get_all_logs.sh]: openebs operator namespace can be different. an optional parameter can be provided to over-ride the default of openebs.

File: debug-scripts/get_all_logs.sh [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+		echo "Print pool pod "$pool_pod" previous instance logs:"
+		kubectl logs $pool_pod -c cstor-pool -n $operator_ns -p
+	done
+}
+
+if [ $# -ne 3 ]; then
+	usage
+fi
+
```
Comment (kmova) [inline debug-scripts/get_all_logs.sh]: Might be a good idea to list the details of the custom resources also -- like disks, sp, spc, csp, cv, cvr and also storgeclasses.

Comment (vishnuitta) [reply debug-scripts/get_all_logs.sh]: makes sense Kiran.. can this be extended with other PRs?

File: debug-scripts/get_all_logs.sh (hunk 1 above)
Comment (vishnuitta) [reply debug-scripts/get_all_logs.sh]: to make it to give minimal options, is there a way to find the operator namespace by using some kubectl commands?

**Comment by vishnuitta** [comment]
yes @ksatchit .. will change the directory path

**Comment by ksatchit** [comment]
Taking script in current form to aid repo migration. Thanks !

**Pull request merged**